src/main/java/org/nest/units/unitrepresentation/SIData.java

```diff
@@ -61,7 +61,7 @@
   private static String[] SIUnitsDerivedRaw =
       {"Hz","N","Pa","J","W","C","V","F","Ohm","S","Wb","T","H","lm","lx","Bq","Gy","Sv","kat"};
   private static String[] SIPrefixesRaw =
-      {"da","h","k","M","G","T","P","E","Z","Y","d","c","m","mu","n","p","f","a","z","y"};
+      {"k","M","G","T","P","E","Z","Y","m","mu","n","p","f","a","z","y"};
```

Comment:
I agree, not even sure what "da" stands for. Not entirely happy with "mu", but I guess there is no alternative if we want to stick to standard ASCII.